cloud_storage/cloud.py

```diff
@@ -25,57 +17,75 @@ def __init__(self):
         '''
         self.sections = config.sections()
 
-    def upload_to_cloud(self, blob_Path, local_Path, file_Name):
-        '''
-        Function to upload the file to respective cloud based on the available section in the config file
 
-        :param: blob_Path - The path where the file would be stored in the cloud 
+        
+    def upload_to_cloud(self, filesList ,folderPathName, local_Path):
```

Comment:
@adithyadinesh0412 function should accept array of files as input, We can implement this in next release. We need to make note of this